docs/legacymode.md

```diff
@@ -0,0 +1,42 @@
+
+The setting can be reverted by returning to the same property sheet menu and unchecking the box then pressing `OK`.
+
+**NOTE:** This setting is globally applied to all sessions that start after the preference is changed. Sessions that are already open will not be changed.
+
+## Differences between modes
+
+The Console Host team strives to minimize differences between the Legacy and current modes of the console to ensure that as many customers as possible can run the most up-to-date version. If you experience an issue that requires you to use the legacy console that is not documented here, please contact the team on the [Microsoft/Terminal](https://github.com/microsoft/terminal/) GitHub repository or via the [Feedback Hub](https://docs.microsoft.com/windows-insider/feedback-hub/feedback-hub-app) for assistance.
+
+### 16-bit applications on 32-bit Windows
+
+Some 16-bit applications on 32-bit Windows use a virtual machine technology to operate called [NTVDM](https://docs.microsoft.com/windows/compatibility/ntvdm-and-16-bit-app-support). Often these applications use a graphical screen buffering mode in conjunction with the console hosting environment to operate. Only the legacy console experience supports these graphical buffering modes and the additional console API support required to power these applications. The system will automatically select the legacy console environment when one of these applications is launched.
+
+### IME Embedding
+
+The legacy Console Host embedded the suggestion portion of the IME inside the hosting window by reserving a line at the bottom of the screen for suggestions. The current Console Host environment instead delegates this activity to the IME subsystem to display an overlay window above the console host with suggestions. In an environment where overlay windows are not possible (like with certain remoting tools), the legacy console host may be required.
+
+### API Differences
+
+The major known difference between legacy and current is the implementation of UTF-8. The legacy host has extremely rudimentary and often incorrect support of UTF-8 with [code page 65001](https://docs.microsoft.com/windows/win32/intl/code-pages). The current console host contains incremental improvements release-over-release of Windows 10 to improve this support. Applications that are attempting to rely on predicting "known incorrect" interpretations of UTF-8 from the legacy console will find themselves receiving different answers as support is improved. 
+
+Other differences experienced with APIs should be reported to [Microsoft/Terminal](https://github.com/microsoft/terminal/) GitHub repository or via the [Feedback Hub](https://docs.microsoft.com/windows-insider/feedback-hub/feedback-hub-app) for triage and possible remediation.
```

Comment:
Same as above, should we just link the new issues page to save an extra step?

Reply:
resolving per Dustin's feedback